boards/arm/nrf5340_audio_dk_nrf5340/board.cmake

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP_NS)
+  set(TFM_PUBLIC_KEY_FORMAT "full")
+endif()
+
+if(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP OR CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP_NS)
+board_runner_args(jlink "--device=nrf5340_xxaa_app" "--speed=4000")
+endif()
+
+if(CONFIG_TFM_FLASH_MERGED_BINARY)
+  set_property(TARGET runners_yaml_props_target PROPERTY hex_file "${CMAKE_BINARY_DIR}/tfm_merged.hex")
+endif()
```

Comment:
So, thingy53 does not have it, we also do not have TF-M support for this board.
This if will never be invoked, and can be removed.

Not a big deal though, just thinking of consistency.
```suggestion
```